resources/icons/spinner.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="Layer_1" data-name="Layer 1" viewBox="0 0 80 80">
```

Comment:
Do we need this separate SVG? how is this implemented in shoreditch?

Reply:
There is no spinner animation present in Shoreditch (as far as styleguide is concerned). Jon suggested to use the svg used in civihr http://1.test.civihr.net/sites/all/modules/civicrm/tools/extensions/civihr/org.civicrm.bootstrapcivihr/img/spinner.svg.